core/src/costs.rs

```diff
@@ -332,6 +357,8 @@ impl RuntimeCosts {
             ReplyPushInput(len) => s
                 .gr_reply_push_input
                 .saturating_add(s.gr_reply_push_input_per_byte.saturating_mul(len.into())),
+            ReplyInput(len) => ReplyPushInput(len).token(s).add(ReplyCommit.token(s)),
+            SendInput(len) => SendPushInput(len).token(s).add(SendCommit(0).token(s)),
```

Comment:
send init also

Reply:
but open an issue cause this approach of calculations still wrong, due to the fact the send input will include 6 host fns instead of 2

Reply:
https://github.com/gear-tech/gear/issues/2255